src/network/serverpackethandler.cpp

```diff
@@ -52,11 +52,12 @@ void Server::handleCommand_Init(NetworkPacket* pkt)
 	if(pkt->getSize() < 1)
 		return;
 
-	RemoteClient* client = getClient(pkt->getPeerId(), CS_Created);
+	session_t peer_id = pkt->getPeerId();
+	RemoteClient* client = getClient(peer_id, CS_Created);
```

Comment:
Please correct the pointer placement for all touched lines. Most common one: `RemoteClient* `

Reply:
I've changed the asterisk position for the four RemoteClient pointers.